### DISCUSSION TITLE
Three QoL requests: Please add queues, mirror Youtube hotkeys (space to pause/play videos), and when autoplaying videos in fullscreen stay in fullscreen on video change

### DISCUSSION THREAD

**Discussion by rossjohnson87** [FreeTubeApp/FreeTube#4599] [General]
Love Freetube, these are my biggeset gripes with it. Think these would a go a long way towards improving the application.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
These requests are already documented and is a duplicate of #547 #2295 #2138
